src/Hydrators/CountryHydrator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace TheRealMVP\Hydrators;
+use TheRealMVP\Entities\Country;
+
+class CountryHydrator
+{
+    private \PDO $pdoConnection;
+
+    /**
+     * TeamHydrator constructor - saves PDO connection object as local variable
+     *
+     * @param $pdoConnection
+     */
+    public function __construct(\PDO $pdoConnection)
+    {
+        $this->pdoConnection = $pdoConnection;
+    }
+
+    /**
+     * create database connection and retrieves data and returns an array of country objects
+     *
+     * @param \PDO $pdoConnection
+     *
+     * @return array
+     */
+    public static function getData(\PDO $pdoConnection) : array
```

Comment:
We now have to give this a PDO connection two times, no need to pass it in as a param for the getData method since we already have it as a property

Reply:
Removed constructor function